### PR TITLE
Eg final namespaces ga

### PR DESCRIPTION
**Pull request by billwert**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/eventgrid/azure-messaging-eventgrid-namespaces/CHANGELOG.md [hunk 1]
```diff
@@ -1,16 +1,18 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0 (Unreleased)
 
 ### Features Added
+- Initial GA release
 
-### Breaking Changes
+## 1.0.0-beta.1 (Unreleased)
```
Comment (mssfang) [inline sdk/eventgrid/azure-messaging-eventgrid-namespaces/CHANGELOG.md:8]: this should be removed.

File: sdk/eventgrid/azure-messaging-eventgrid-namespaces/assets.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-  "AssetsRepo": "Azure/azure-sdk-assets",
-  "AssetsRepoPrefixPath": "java",
-  "TagPrefix": "java/eventgrid/azure-messaging-eventgrid-namespaces",
-  "Tag": "java/eventgrid/azure-messaging-eventgrid-namespaces_0ec3de5f92"
-}
+  "AssetsRepo" : "Azure/azure-sdk-assets",
+  "AssetsRepoPrefixPath" : "java",
+  "TagPrefix" : "java/eventgrid/azure-messaging-eventgrid-namespaces",
+  "Tag" : ""
```
Comment (mssfang) [inline sdk/eventgrid/azure-messaging-eventgrid-namespaces/assets.json:5]: missing tag value

Comment (billwert) [reply sdk/eventgrid/azure-messaging-eventgrid-namespaces/assets.json:5]: Yeah, this is very in-progress. Will get it cleaned up and re-tag you.

File: sdk/eventgrid/azure-messaging-eventgrid-namespaces/pom.xml [hunk 3]
```diff
@@ -14,11 +14,11 @@
 
   <groupId>com.azure</groupId>
   <artifactId>azure-messaging-eventgrid-namespaces</artifactId>
-  <version>1.0.0-beta.2</version> <!-- {x-version-update;com.azure:azure-messaging-eventgrid-namespaces;current} -->
+  <version>1.0.0</version> <!-- {x-version-update;com.azure:azure-messaging-eventgrid-namespaces;current} -->
   <packaging>jar</packaging>
 
   <name>Microsoft Azure SDK for EventGrid</name>
```
Comment (mssfang) [inline sdk/eventgrid/azure-messaging-eventgrid-namespaces/pom.xml:20]: do we need to update the name tag here to add `namespace` at the end?

**Review by mssfang — APPROVED** [review]

**Pull request merged**